src/index.js

```diff
@@ -2,6 +2,7 @@
 export {default as run} from './lib';
 export {default as cli} from './cli';
 export {default as getPackages} from './utils/getPackages';
+export {default as filterWorkspaces} from './utils/filterWorkspaces';
```

Comment:
We should not expose this function so that it requires a `Workspace` object to be passed

Reply:
No worries, will remove for now.

Do you think it'd be worth splitting the utils into ones that are exposed and ones that arent, and having the ones that are exposed give you the simplified output like we do with getPackages?